msg_kitti_object[] objects
geometry_msgs/Quaternion location
geometry_msgs/Vector3 linear_velocity
geometry_msgs/Vector3 linear_acceleration
float32[] x
---
float32 par